walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java

```diff
@@ -270,5 +272,155 @@ public void notEnoughRoom() {
 
     Assert.assertEquals(expected, actual);
   }
-}
 
+  @Test
+  public void optinalNoImpact() {
```

Comment:
Can you rename your tests to be more descriptive? For instance, looking at this I have no idea what "impact" means in this context.